spec/models/form/sales/pages/property_local_authority_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
```

Comment:
Hm, maybe needs a rebase onto the main? Cause I think this change is on a different branch as well as `spec/models/form/sales/questions/property_local_authority_spec.rb`